third_party/2/six/__init__.pyi

```diff
@@ -27,7 +27,7 @@ PY2 = True
 PY3 = False
 PY34 = False
 
-string_types = basestring,
+string_types = Union[str, unicode],
```

Comment:
This would change the length of `six.string_types` in Python 2: it's currently a tuple of length one in Python 2 and length two in Python 3. I'm fine with making the change to `string_types = (str, unicode)`, though AFAICT, they do the same thing in this case.